Makefile

```diff
@@ -81,6 +81,7 @@ dev-switch:
 	test "$(shell opam switch show)" = "$(shell pwd)" || \
 		opam switch create -y . $(TEST_OCAMLVERSION) --deps-only --with-test
 	opam install -y --update-invariant ocaml.$(TEST_OCAMLVERSION) $(TEST_DEPS) $(DEV_DEPS)
+	opam pin add -y coq-of-ocaml https://github.com/formal-land/coq-of-ocaml/tree/ocaml-4.14
```

Comment:
This is my way of pointing to a specific .opam file of a specific branch. I am not sure if there is a better way to do this.